tests/test_pyconfyg.py

```diff
@@ -0,0 +1,19 @@
+from pyconfyg import PyConfyg, Confyg, parse_strings
+
+import io
+
+def test_parse_strings():
+    assert parse_strings("a=2") == {'a': 2}
+
+def test_parse_strings2():
+    assert parse_strings("a=2", "b=4") == {'a': 2, 'b': 4}
+
+def test_parse_strings3():
```

Comment:
And rename this to `test_parse_strings_with_env`

Reply:
I would also add a test case where the string overrides the content of the `env`:
```py
env = {"a":  2}
assert parse_strings("a=1", env=env) == {"a": 1}
```